lib/python/rose/bush.py

```diff
@@ -452,6 +459,7 @@ def view(self, user, suite, path, path_in_tar=None, mode=None):
             rose_version=self.rose_version,
             script=cherrypy.request.script_name,
             time=strftime("%Y-%m-%dT%H:%M:%S+0000", gmtime()),
+            logo=self.logo,
```

Comment:
I think this repeated setting of self.logo is once again highlighting the levels of redundancy in the setting of these data dicts.